common/com/twitter/intellij/pants/model/PantsVersion.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package com.twitter.intellij.pants.model;
+
+import com.twitter.intellij.pants.PantsException;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PantsVersion {
+
+  private int major;
+  private int minor;
+  private int patch;
+  private PantsPatchVersionKind kind;
+
+  // Link to regular expression cehcking this: https://regex101.com/r/SUZCl6/2
```

Comment:
```suggestion
  // Link to regular expression checking this: https://regex101.com/r/SUZCl6/2
```

Reply:
Btw, *fantastic* move to link to regex101.com -- thanks for thinking about this! This is exactly how we make regex a little more comprehensible.